doc/sphinx/changes.rst

```diff
@@ -8,6 +8,613 @@ Recent changes
+  (`#14819 <https://github.com/coq/coq/pull/14819>`_,
+  by Pierre Roux).
+- **Deprecated:**
+  ``frexp`` and ``ldexp`` in `FloatOps.v`, renamed ``Z.frexp`` and ``Z.ldexp``
+  (`#15085 <https://github.com/coq/coq/pull/15085>`_,
+  by Pierre Roux).
+- **Added:**
+  A proof that incoherent equivalences can be adjusted to adjoint equivalences in ``Logic.Adjointification``.
+  (`#13408 <https://github.com/coq/coq/pull/13408>`_,
+  by Jasper Hugunin).
+- **Added:**
+  ``ltb`` and ``leb`` functions for ``string``, and some lemmas about them;
+- **Added:**
+  simple non dependent product ``slexprod`` in
+  ``Relations/Relation_Operators.v`` and its proof of well-foundness
+  ``wf_slexprod`` in ``Wellfounded/Lexicographic_Product.v``
+  (`#14809 <https://github.com/coq/coq/pull/14809>`_,
+  by Laurent Thery).
+- **Added:**
+  The notations ``(x; y)``, ``x.1``, ``x.2`` for sigT are now exported and available  after ``Import SigTNotations.``.
+  (`#14813 <https://github.com/coq/coq/pull/14813>`_, by Laurent Théry).
+- **Added:**
+  The function ``sigT_of_prod`` turns a pair ``A * B`` into ``{_ : A & B}``. Its inverse function is ``prod_of_sigT``. This is shown by theorems ``sigT_prod_sigT`` and ``prod_sigT_prod``.
+  (`#14813 <https://github.com/coq/coq/pull/14813>`_, by Laurent Théry).
+- **Fixed:**
+  ``split_combine`` lemma for lists, making it usable
+  (`#14458 <https://github.com/coq/coq/pull/14458>`_,
+  by Yishuai Li).
+
+Infrastructure and dependencies
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+- **Changed:**
+  Coq's continuous integration now provides a more accessible Windows
+  installer artifact in the "Checks" GitHub tab, both for pull
+  requests and the `master` branch.
+
+  This facilitates testing Coq's bleeding edge builds on Windows, and
+  should be more reliable than the previous setup (`#12425
+  <https://github.com/coq/coq/pull/12425>`_, by Emilio Jesus Gallego
+  Arias).
+- **Changed:**
+  Coq's ``./configure`` script has gone through a major cleanup. In
+  particular, the following options have been removed:
+  - ``-force-caml-version``, ``-force-findlib-version``: Coq won't
+  compile with OCaml or findlib lower than the required versions;
+  - ``-vmbyteflags``, ``-custom``, ``-no-custom``: linking options for
+  toplevels are now controlled in ``topbin/dune``;
+  - ``-ocamlfind``: Coq will now use the toolchain specified in the
+  Dune configuration; this can be controlled using the workspaces
+  feature;
+  - ``-nodebug``: Coq will now follow the standard, which is to always
+  pass ``-g`` to OCaml; this can be modified using a custom Dune workspace;
+  - ``-flambda-opts``: compilation options are now set in Coq's root
+  ``dune`` file, can be updated using a custom Dune workspace;
+  - ``-local``, ``-bindir``, ``-coqdocdir``, ``-annotate``,
+  ``-camldir``, ``-profiling``: these flags were deprecated in 8.14,
+  and are now removed.
```

Comment:
This was intended to be a bullet-point list but it doesn't work. I'm not sure what is needed to make it work. Probably newlines before and after the bullet-point list and indentation of bulleted paragraphs.

Reply:
let's see how it looks now

Reply:
Looks good: https://coq.gitlab.io/-/coq/-/jobs/1846813995/artifacts/_install_ci/share/doc/coq/sphinx/html/changes.html#id156